CoolestMovieAPI/CoolestMovieAPI/DTO/DirectorDTO.cs

```diff
@@ -1,17 +1,23 @@
 ﻿using CoolestMovieAPI.Models;
 using System;
 using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
 using System.Linq;
 using System.Threading.Tasks;
 
 namespace CoolestMovieAPI.DTO
 {
     public class DirectorDTO
     {
-        public int Id { get; set; }
-        public string Name { get; set; }
-        public DateTime BirthDate { get; set; }
-        public string Country { get; set; }
-        public Dictionary<string, Movie> Directed { get; set; }
+        [Required]
+        public int DirectorId { get; set; }
+        [Required]
+        public string DirectorName { get; set; }
+        [Required]
```

Comment:
I added a movie to the database where I was unable to find the birthdate of the Director, So is this really required?

Reply:
You are right, I have removed the annotations.